migration/i7.1z/oracle/202009161600_IDEMPIERE-3336.sql

```diff
@@ -0,0 +1,132 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- 08/04/2017 11h23min15s BRT
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,Description,Help,AD_Table_ID,ColumnName,DefaultValue,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,SeqNo,IsEncrypted,AD_Reference_ID,AD_Reference_Value_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,ReadOnlyLogic,EntityType,IsSyncDatabase,IsAlwaysUpdateable,IsAutocomplete,IsAllowLogging,AD_Column_UU,IsAllowCopy,SeqNoSelection,IsToolbarButton,IsSecure,FKConstraintType) VALUES (212979,0,'Entity Type','Dictionary Entity Type; Determines ownership and synchronization','The Entity Types "Dictionary", "iDempiere" and "Application" might be automatically synchronized and customizations deleted or overwritten.
+
+For customizations, copy the entity and select "User"!',200108,'EntityType','U',40,'N','N','N','N','N',0,'N',18,389,0,0,'Y',TO_DATE('2017-04-08 11:23:15','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2017-04-08 11:23:15','YYYY-MM-DD HH24:MI:SS'),100,1682,'Y','N','@EntityType@=D','D','N','N','N','Y','b70c3127-a44f-4b93-8ed7-8c7782cae0af','Y',0,'N','N','N')
+;
+
+-- 08/04/2017 11h23min15s BRT
+INSERT INTO AD_Column_Trl (AD_Language,AD_Column_ID, Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,AD_Column_Trl_UU ) SELECT l.AD_Language,t.AD_Column_ID, t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,Generate_UUID() FROM AD_Language l, AD_Column t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Column_ID=212979 AND NOT EXISTS (SELECT * FROM AD_Column_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Column_ID=t.AD_Column_ID)
+;
+
+-- 08/04/2017 11h23min19s BRT
+UPDATE AD_Column SET FKConstraintName='EntityType_ADStatusLineUsedIn', FKConstraintType='N',Updated=TO_DATE('2017-04-08 11:23:19','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=212979
+;
+
+-- 08/04/2017 11h23min19s BRT
+ALTER TABLE AD_StatusLineUsedIn ADD EntityType VARCHAR2(40) DEFAULT 'U'
```

Comment:
For oracle it must use VARCHAR2(40 CHAR)

Reply:
:disappointed: I need to be more careful with Oracle scripts